docs/src/manual/power-flow.md

```diff
@@ -0,0 +1,105 @@
+
+
+## Generic Power Flow
+
+The general purpose power flow solver in PowerModels is,
+
+```@docs
+solve_mc_pf
+```
+
+This function builds a JuMP model for a wide variety of the power flow formulations supported by PowerModelsDistribution.  For example it supports,
+* `ACPUPowerModel` - a non-convex nonlinear AC power flow using complex voltages in polar coordinates
+* `IVRUPowerModel` - a non-convex nonlinear AC power flow using current voltage variables in rectangular coordinates
+* `AbstractUBFModels` - a non-convex nonlinear branch flow power flow  <!-- TODO type?
+
+
+The `solve_mc_pf` solution method is both formulation and solver agnostic and can leverage the wide range of solvers that are available in the JuMP ecosystem.  Many of these solvers are commercial-grade, which in turn makes `solve_mc_pf` the most reliable power flow solution method in PowerModelsDistribution.
+
+<!-- !!! note
+    Use of `solve_mc_pf` is highly recommended over the other solution methods for increased robustness.  Applications that benefit from the Julia native solution methods are an exception to this general rule. -->
+
+
+### Warm Starting
+
+In some applications an initial guess of the power flow solution may be available.  In such a case, this information may be able to decrease a solver's time to convergence, especially when solving systems of nonlinear equations.  The `_start` postfix can be used in the network data to initialize the solver's variables and provide a suitable solution guess.  The most common values are as follows,
+
+For each generator,
+* `pg_start` - active power injection starting point
+* `qg_start` - reactive power injection starting point
+
+For each bus,
+* `vm_start` - voltage magnitude starting point for the `ACPUPowerModel` model
+* `va_start` - voltage angle starting point for the `ACPUPowerModel` model
+* `vr_start` - real voltage starting point for the `IVRUPowerModel` model
+* `vi_start` - imaginary voltage starting point for the `IVRUPowerModel` model
+
+The following helper function can be used to use the solution point in the network data as the starting point for `solve_mc_pf`.  <!-- TODO correct?
+```@docs
+add_start_voltage!
+```
+
+!!! warning
+    Warm starting a solver is a very delicate task and can easily result in degraded performance.  Using PowerModelsDistribution' default flat-start values is recommended before experimenting with warm starting a solver.
+
+
+## Native Power Flow
+
+The AC Power Flow problem is ubiquitous in power system analysis. The problem requires solving a system of nonlinear equations, usually via a Newton-Raphson type of algorithm.  In PowerModelsDistribution, the standard Julia library is used for solving this system of nonlinear equations.  The following function is used to solve Power Flow problem with voltages in polar coordinates.
```

Comment:
AC Unbalanced Power Flow problem...

Reply:
rectangular coordinates

Reply:
probably mention current injection method here, and that it is inspired by OpenDSS's algorithm